File: 프로그래머스/1/12932. 자연수 뒤집어 배열로 만들기/자연수 뒤집어 배열로 만들기.py
def solution(n):
    answer = []
    #for i in str(n) :
    #    answer.append(int(i))
    #answer.reverse()
    print(str(n))
    n = str(n)
    for i in range(-1,-len(n)-1,-1) :
        numb = str(n)[i]
        print(numb)
        answer.append(int(numb))
    return answer

File: 프로그래머스/1/12932. 자연수 뒤집어 배열로 만들기/자연수 뒤집어 배열로 만들기.java
class Solution {
    public int[] solution(long n) {
        
        String strn = String.valueOf(n);
        int len = strn.length();
        int[] answer = new int[len];
        
        for(int i =0; i<len; i++){
            answer[i] = strn.charAt(len-1-i) -'0';
        }
        
        return answer;
    }
}